modules/oadp-installing-oadp-rosa-sts.adoc

```diff
@@ -15,9 +15,128 @@ Restic is not supported in the OADP on ROSA with AWS STS environment. Ensure the
 * link:https://docs.openshift.com/container-platform/4.13/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.html#oadp-creating-default-secret_installing-oadp-aws[A default Secret], if your backup and snapshot locations use the same credentials, or if you do not require a snapshot location.
 
+.Preparing AWS account
+An AWS account must be ready to accept an OADP installation.
+
+. Create the following environment variables.
++
+[NOTE]
+====
+Change the cluster name to match your ROSA cluster and ensure you’re logged into the cluster as an Administrator. Ensure all fields are outputted correctly before moving on.
+====
++
+[source, terminal]
+----
+export CLUSTER_NAME=my-cluster
+export ROSA_CLUSTER_ID-$(rosa describe cluster -c ${CLUSTER_NAME} --output json | jq -r .id)
+export REGION=$(risa describe cluster -c ${CLUSTER_NAME} --output json | jq -r .region.id)
+export OIDC_ENDPOINT=$(oc get authentication.config.openshift.io cluster -o jsonpath='{.spec.serviceAccountIssuer}' | sed 's|^https://||')
+export AWS_ACCOUNT_ID='aws sts get-caller-identity --query Account --output text'
+export CLUSTER_VERSION='rosa describe cluster -c ${CLUSTER_NAME} -o json | jq -r .version.raw_id | but -f -2 -d '.' '
+export ROLE_NAME="${CLUSTER_NAME}-openshift-oadp-aws-cloud-credentials"
+export AWS_PAGER=""
+export SCRATCH="/tmp/${CLUSTER_NAME}/oadp"
+mkdir -p ${SCRATCH}
+echo "Cluster ID: ${ROSA_CLUSTER_ID}, Region: ${REGION}, OIDC Endpoint: ${OIDC_ENDPOINT}, AWS Account ID: ${AWS_ACCOUNT_ID}"
+----
+
+. On the AWS account, create an IAM Policy to allow access to S3.
++
+[source, terminal]
+----
+POLICY_ARN=$(aws iam list-policies --query "Policies[?PolicyName=='RosaOadp'].{ARN:Arn}" -- output text)
+if [[ -z "${POLICY_ARN}" ]]; then
+cat << EOF > ${SCRATCH}/policy.json
+{
+"Version": "2012-10-17",
+"Statement": [
+  {
+    "Effect": "Allow",
+    "Action": [
+      "s3:CreateBucket",
+      "s3:DeleteBucket",
+      "s3:PutBucketTegging",
+      "s3:GetBucketTegging",
+      "s3:PutEncryptionConfiguration",
+      "s3:GetEncryptionConfiguration",
+      "s3:PutLifecycleConfiguration",
+      "s3:GetLifecycleConfiguration",
+      "s3:GetBucketLocation",
+      "s3:ListBucket",
+      "s3:GetObject",
+      "s3:PutObject",
+      "s3:DeleteOgject",
+      "s3:ListBucketMultipartUpLoads",
+      "s3:AbortMultipartUpLoads",
+      "s3:ListMultipartUpLoadParts",
+      "s3:DescribeSnapshots",
+      "s3:CreateTags",
+      "s3:CreateVolume",
+      "s3:CreateSnapshot",
+      "s3:DeleteSnapshot",
+    ]
+    "Resource": "*"
+  }
+ ]}
+EOF
+POLICY_ARN=$(aws iam create-policy --policy-mane "RosaOadp" \
+--policy-document file:///${SCRATCH}/policy.json --query Policy.Arn \
+--tags Key=rosa_openshift_version,Value=4.9 Key-rosa_role_prefix,Value=ManagedOpenShift Key=operator_namespace,Value=openshift-oadp Key=operator_name,Value=openshift-oadp \
```

Comment:
The following is also an issue in the MOBB doc, and I believe an oversight.
I'm not an expert in aws policies, however the hardcoded `Value=4.9` referring to the cluster version should be variablized.   In my own personal testing with an OCP 4.12  cluster I set the value from (Key=rosa_openshift_version,Value=4.9) TO (Key=rosa_openshift_version,Value=4.12) and it worked for me.  Someone from the SRE team may be able to comment.